ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+open class ContactView: UIView {
+    private struct Constants {
+        static let contactWidth: CGFloat = 70.0
+        static let avatarViewWidth: CGFloat = 70.0
+        static let avatarViewHeight: CGFloat = 70.0
+        static let titleLabelMinimumHeight: CGFloat = 12.0
+        static let spacingBetweenAvatarAndFirstLabel: CGFloat = 13.0
+    }
+
+    // TODO: Set it as the default image (pawn)
+    @objc public var avatarImage: UIImage? {
+        didSet {
+            if let avatarImage = avatarImage {
+                avatarView.setup(image: avatarImage)
+            }
+        }
+    }
+    private let avatarView: AvatarView
+    private var titleLabel: UILabel?
+    private var subtitleLabel: UILabel?
+
+    /// Initializes the contact view by creating an avatar view with a first name and last name
+    ///
+    /// - Parameters:
+    ///   - title: String that will be the text of the top label
+    ///   - subtitle: String that will be the text of the bottom label
+    @objc public init(title: String, subtitle: String) {
+        avatarView = AvatarView(avatarSize: .extraExtraLarge, withBorder: false, style: .circle)
+        titleLabel = UILabel()
+        subtitleLabel = UILabel()
+        super.init(frame: .zero)
+        setupAvatarView(with: title, and: subtitle, or: nil)
+        backgroundColor = Colors.surfacePrimary
+        setupTitleLabel(using: title)
+        setupSubtitleLabel(using: subtitle)
+        setupLayout()
+    }
+
+    /// Initializes the contact view by creating an avatar view with an identifier
+    ///
+    /// - Parameters:
+    ///   - identifier: String that will be used to identify the contact (e.g. email, phone number, first name)
+    @objc public init(identifier: String) {
+        avatarView = AvatarView(avatarSize: .extraExtraLarge, withBorder: false, style: .circle)
```

Comment:
You don't need titleLabel = UILabel() here?

Reply:
Right, I made the correction when making changes for https://github.com/microsoft/fluentui-apple/pull/101#discussion_r444592675